batch/sql/rm-resource-names-agg-resources.sql

```diff
@@ -0,0 +1,92 @@
+ALTER TABLE attempt_resources DROP PRIMARY KEY, ADD PRIMARY KEY (batch_id, job_id, attempt_id, resource_id), ALGORITHM=INPLACE, LOCK=NONE;
+ALTER TABLE attempt_resources MODIFY resource VARCHAR(100), ALGORITHM=INPLACE, LOCK=NONE;
```

Comment:
What does this `MODIFY` here do?

Reply:
Changes the column type to be allowed to be NULL. Right now it can't be null.